tests/python/pytorch/graphbolt/test_subgraph_sampler.py

```diff
@@ -477,35 +473,36 @@ def test_SubgraphSampler_Random_Hetero_Graph(sampler_type):
     fanouts = [torch.LongTensor([2]) for _ in range(num_layer)]
     sampler = _get_sampler(sampler_type)
 
-    sampler_dp = sampler(item_sampler, graph, fanouts, replace=True)
+    sampler_dp = sampler(item_sampler, graph, fanouts)
```

Comment:
why remove `replace`? Are both `replace=False` and `replace=True` covered in tests?

Reply:
Sampling with replacement not yet supported on GPU. Now both `replace=False` and `replace=True` are covered in cpu sampling tests.